Sources/PaymentsCore/Networking/Enums/CoreError.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+// TODO: Refine how we want to do error handling
+/// For further reading, see https://developer.apple.com/documentation/swift/error
+/// For Discussion:
+/// Error conforming enum vs Error conforming Struct?
```

Comment:
I'm curious to hear some BT thoughts on this as I think errors are handled in BT as a whole a lot better than the PayPal SDKs! I think a Struct will give us a bit more flexibility to potentially return more details in the error.

Reply:
Gonna check out this article & come back to this comment!

Reply:
All good if we punt on this decision for now and handle as a follow up work item?